SECURITY.md

```diff
@@ -0,0 +1,14 @@
+# Security Policy
+
+## Supported Versions
+
+Use this section to tell people about which versions of your project are
+currently being supported with security updates.
+
+| Version   | Supported          |
+| --------- | ------------------ |
+| 0.1.x     | :white_check_mark: |
```

Comment:
Are you referring to a real version here?

Reply:
No, this is just a placeholder version for the example